app/templates/frameworks/edit_declaration_section.html

```diff
@@ -33,12 +33,9 @@
 
 {% block mainContent %}
   {% if name_of_framework_that_section_has_been_prefilled_from %}
-    {% with
-       message = "Answers on this page are from an earlier declaration and need review.",
-       type = "information"
```

Comment:
Is the `type` important? We didn't keep it in a [similar PR](https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/1056) for the buyer-frontend

Reply:
No, the banner component doesn't use type. Our alert component uses it for styling success messages green, etc.